lib/ts/recipe/thirdpartyemailpassword/recipe.ts

```diff
@@ -48,9 +48,9 @@ export default class Recipe extends RecipeModule {
 
     emailVerificationRecipe: EmailVerificationRecipe;
 
-    private emailPasswordRecipe: EmailPasswordRecipe;
+    emailPasswordRecipe: EmailPasswordRecipe;
 
-    private thirdPartyRecipe: ThirdPartyRecipe | undefined;
+    thirdPartyRecipe: ThirdPartyRecipe | undefined;
```

Comment:
I think if you change the constructor of the backward service to not take a recipe, you can make these back to private.